switch/ffmpeg/fix_net.patch

```diff
@@ -0,0 +1,24 @@
+diff -ura pkg.orig/libavformat/os_support.c pkg.new/libavformat/os_support.c
```

Comment:
Is there any way that libnx could be enhanced so that this patch is rendered unnecessary?

Reply:
idk about the `HAVE_INET_ATON` patch but a lot of software assumes that `poll.h` is in `sys/poll.h` so I guess changing that would help

Reply:
>I'm not keen on -Ofast, it has interesting side effects. Have you tested the impact at all?

I haven't tested the impact of Ofast, just enabled it and noticed that none of the stuff I was using broke, I probably shouldn't have included it in the pr.